src/components/KuadrantPoliciesPage.tsx

```diff
@@ -0,0 +1,422 @@
+      id: 'Status',
+    },
+    {
+      title: t('plugin__console-plugin-template~Created'),
+      id: 'Created',
+      sort: 'metadata.creationTimestamp',
+      transforms: [sortable],
+    },
+    {
+      title: '',  // No title for the kebab menu column
+      id: 'dropdown-with-kebab',
+      props: { className: 'pf-v5-c-table__action' },
+    },
+  ];
+
+  const AllPolicyRow: React.FC<RowProps<K8sResourceCommon>> = ({ obj, activeColumnIDs }) => {
+    const [group, version] = obj.apiVersion.includes('/') ? obj.apiVersion.split('/') : ['', obj.apiVersion];
+    return (
+      <>
+        <TableData id={columns[0].id} activeColumnIDs={activeColumnIDs}>
+          <ResourceLink groupVersionKind={{group: group, version: version, kind: obj.kind}} name={obj.metadata.name} namespace={obj.metadata.namespace} />
+        </TableData>
+        <TableData id={columns[1].id} activeColumnIDs={activeColumnIDs}>{obj.kind}</TableData>
+        <TableData id={columns[2].id} activeColumnIDs={activeColumnIDs}>
+          <ResourceLink groupVersionKind={{version: "v1", kind: "Namespace"}} name={obj.metadata.namespace} />
+        </TableData>
+        <TableData id={columns[3].id} activeColumnIDs={activeColumnIDs}>{statusConditionsAsString(obj)}</TableData>
+        <TableData id={columns[4].id} activeColumnIDs={activeColumnIDs}>
+          <Timestamp timestamp={obj.metadata.creationTimestamp} />
+        </TableData>
+        <TableData id={columns[5].id} activeColumnIDs={activeColumnIDs} className="pf-v5-c-table__action">
+          <DropdownWithKebab obj={obj} />
+        </TableData>
+      </>
+    );
+  };
+
+  return (
+    <VirtualizedTable<K8sResourceCommon>
+      data={data}
+      unfilteredData={unfilteredData}
+      loaded={loaded}
+      loadError={loadError}
+      columns={columns}
+      Row={AllPolicyRow}
+    />
+  );
+};
+
+type DropdownWithKebabProps = {
+  obj: K8sResourceCommon;
+};
+
+const DropdownWithKebab: React.FC<DropdownWithKebabProps> = ({ obj }) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = (_event: React.MouseEvent<Element, MouseEvent> | undefined, value: string | number | undefined) => {
+    // eslint-disable-next-line no-console
+    console.log('selected', value);
+    setIsOpen(false);
+  };
+
+  return (
+    <Dropdown
+      isOpen={isOpen}
+      onSelect={onSelect}
+      onOpenChange={(isOpen: boolean) => setIsOpen(isOpen)}
+      toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+        <MenuToggle
+          ref={toggleRef}
+          aria-label="kebab dropdown toggle"
+          variant="plain"
+          onClick={onToggleClick}
+          isExpanded={isOpen}
+        >
+          <EllipsisVIcon />
+        </MenuToggle>
+      )}
+      shouldFocusToggleOnSelect
+    >
+      <DropdownList>
+        <DropdownItem value={0} key="edit">
+          Edit
```

Comment:
no need right now, but we may want to look at using the built-in i8n stuff for strings (Example in the old dashboard view)